packages/lwc-language-server/src/javascript/type-mapping.ts

```diff
@@ -217,17 +217,25 @@ function getMemberProperty(propertyObj: ClassProperty): InternalClassMember | nu
         propertyObj.dataProperty,
     );
 
+    // Note there can only be a getter or only a setter, both are not required.
+    // Use the getter if available, or the setter if there is no getter.  
+    let loc: SourceLocation;
+    if (propertyObj.propertyType === 'accessor') {
+        if (propertyObj.getter) {
+            loc = propertyObj.getter.location
+        } else if (propertyObj.setter) {
+            loc = propertyObj.setter.location
+        }
+     } else {
+        loc = propertyObj?.dataProperty.location
+    }
     return stripKeysWithUndefinedVals({
         name: propertyObj.name,
         type: 'property',
         value,
         decorator: decoratorType,
         doc: propertyObj.__internal__doc,
-        loc: externalToInternalLoc(
-            propertyObj.propertyType === 'accessor'
-                ? propertyObj.getter.location
-                : propertyObj.dataProperty.location
-        ),
+        loc: externalToInternalLoc(loc),
```

Comment:
Locally this project fails to compile for me.  Not sure what's up with that. 

![image](https://github.com/forcedotcom/lightning-language-server/assets/76090802/b2baff50-50c0-47ee-822a-f921c1295649)